internal/log_unix.go

```diff
@@ -17,18 +17,31 @@
 package internal
 
 import (
+	"io/ioutil"
 	"log/syslog"
 
+	"github.com/MFAshby/stdemuxerhook"
 	"github.com/matrix-org/dendrite/setup/config"
 	"github.com/sirupsen/logrus"
 	lSyslog "github.com/sirupsen/logrus/hooks/syslog"
 )
 
-// SetupHookLogging configures the logging hooks defined in the configuration.
+// SetupLogging configures the logging hooks defined in the configuration.
 // If something fails here it means that the logging was improperly configured,
 // so we just exit with the error
-func SetupHookLogging(hooks []config.LogrusHook, componentName string) {
+func SetupLogging(hooks []config.LogrusHook, componentName string) {
```

Comment:
Renaming this function will break building on Windows.

Reply:
Fair. Reverting this.